src/main/java/com/commercetools/sync/inventories/InventorySync.java

```diff
@@ -25,34 +25,36 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
 import java.util.Set;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CompletionStage;
 import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 
+import static com.commercetools.sync.inventories.InventorySyncMessages.CTP_INVENTORY_ENTRY_CREATE_FAILED;
```

Comment:
Because of conflicted review https://github.com/commercetools/commercetools-sync-java/pull/58#discussion_r119598293 Actually I should have asked about the motivation.

But I agree with you. Constants belong to a `Sync` only, so they should stay inside the class. I reverted the changes  d78891654a39fbe5f308d8ba3bf78a9b80e43b24